GVFS/GVFS.UnitTests.Windows/Windows/CommandLine/SparseVerbTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿using GVFS.CommandLine;
+using GVFS.Tests.Should;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace GVFS.UnitTests.Windows.Windows.CommandLine
+{
+    [TestFixture]
+    public class SparseVerbTests
+    {
+        // Shorter name for readability
+        private static char dirSep = Path.DirectorySeparatorChar;
+
+        [TestCase]
+        public void PathCoveredBySparseFolders_RootPaths()
+        {
+            string rootPaths = $"a.txt{SparseVerb.StatusPathSeparatorToken}b.txt{SparseVerb.StatusPathSeparatorToken}c.txt{SparseVerb.StatusPathSeparatorToken}";

```

Comment:
I think that you are using `SparseVerb.StatusPathSeparatorToken` so that if it changes it won't break the tests but I would argue that the tests should __not__ use internal information and should break.  So my suggestion is to have
```
private const char StatusPathSeparatorToken = '\0';
```
in the test.  It will only be one place to change but tests will break when the internals of the SparseVerb change.